cmake.deps/CMakeLists.txt

```diff
@@ -263,8 +283,93 @@ endif()
 if(WIN32)
   include(GetBinaryDeps)
 
-  GetBinaryDep(TARGET wintools
-    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory . ${DEPS_BIN_DIR})
+  ExternalProject_Add(cat
+    URL ${CAT_URL}
+    URL_HASH SHA256=${CAT_SHA256}
+    DOWNLOAD_NO_PROGRESS TRUE
+    DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}
+    DOWNLOAD_NO_EXTRACT TRUE
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E make_directory ${DEPS_BIN_DIR}
+    COMMAND ${CMAKE_COMMAND} -E copy <DOWNLOADED_FILE> ${DEPS_BIN_DIR})
```

Comment:
would `GetBinaryDep` avoid some boilerplate here?

Reply:
A variation of it, but in hindsight I don't think it's needed. A single download should be sufficient. I will investigate properly when I can.

Reply:
Fixed